client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -2,33 +2,29 @@
  * External dependencies
  */
 import React, { FunctionComponent, useState } from 'react';
-import {
-	Button,
-	HorizontalRule,
-	Panel,
-	PanelBody,
-	PanelRow,
-	TextControl,
-} from '@wordpress/components';
+import { Button, Panel, PanelBody, PanelRow } from '@wordpress/components';
+import SearchCard from 'components/search-card';
```

Comment:
Does **extraction** here mean a duplicated calypso-free version of the component that lives inside gutenboarding? Anyway, created an issue here: https://github.com/Automattic/wp-calypso/issues/40323

Will add a `// TODO:` before line 6 in a commit.

Reply:
I think we could go just with regular core text input for now to avoid this Calypso component — the only difference is that we won't have the search icon but that's not a biggie for now:

Figma
<img width="597" alt="image" src="https://user-images.githubusercontent.com/87168/77175285-0e5b6f00-6acb-11ea-883a-6742c1a99a4a.png">

In `master`:
<img width="337" alt="image" src="https://user-images.githubusercontent.com/87168/77175329-2206d580-6acb-11ea-931b-033e9eafffd2.png">

This PR:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/87168/77175349-2d5a0100-6acb-11ea-8052-d4ee002da961.png">

...furthermore, the focus ring and border styles wouldn't be consistent with the rest of the UI.

Reply:
Reverting to TextControl. SearchCard was also failing typechecks which can't be fixed within gutenbubble.